tests/foreman/cli/test_errata.py

```diff
@@ -495,6 +495,48 @@ def test_positive_list_affected_chosts(self):
                 virtual_machine.hostname in result
             ), "VM host name not found in list of applicable hosts"
 
+    @tier3
+    def test_install_errata_to_one_host(self):
+        """Install an erratum to one of the hosts in a host collection.
+
+        :id: bfcee2de-3448-497e-a696-fcd30cea9d33
+
+        :expectedresults: Errata was successfully installed in only one of the hosts in
+        the host collection
```

Comment:
Travis complains about this line, says it should be indented.
When trying to fix, you can `make test-docstrings` in your branch to get feedback faster than pushing and waiting for travis.

Reply:
@mirekdlugosz thanks for the tip, `make test-docstrings` found one other error.